spec/models/vote_spec.rb

```diff
@@ -16,81 +16,172 @@
     it "receives vote operation from user" do
       allow(Vote).to receive(:process_vote).and_return (nil)
       expect(Vote.act_on_vote(operation, subject, subject_id, user)).to be nil
-     end
+    end
   end
 
   describe ".process_vote" do
-   it "returns an integer value after vote is processed" do
-    allow(Vote).to receive(:subject_exists?).and_return (true)
-    allow(Vote).to receive(:store_vote).and_return (true)
-    expect(Vote.process_vote(subject_class, subject_id, user, value)).to be_an Integer
-    expect((Vote.process_vote(subject_class, subject_id, user, value).abs)).to eq(1)
-   end
+    it "returns an integer value after vote is processed" do
+      allow(Vote).to receive(:subject_exists?).and_return (true)
+      allow(Vote).to receive(:store_vote).and_return (7)
```

Comment:
Still curious about this **7** cause I think since `:store_vote` return value isn't assigned, _stubbing_ with `nil` won't hurt :weary: